src/test/java/org/glovo/service/OrderServiceTest.java

```diff
@@ -0,0 +1,199 @@
+package org.glovo.service;
+
+import org.glovo.entity.GlovoOrder;
+import org.glovo.entity.Product;
+import org.glovo.repository.OrderRepository;
+import org.glovo.repository.ProductRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.*;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+public class OrderServiceTest {
+
+    @Mock
+    private ProductRepository productRepository;
+
+    @Mock
+    private OrderRepository orderRepository;
+
+    @Mock
+    private ProductService productService;
+
+    @InjectMocks
+    private OrderService orderService = new OrderService();
+    @Captor
+    private ArgumentCaptor<GlovoOrder> orderCaptor;
+
+    @Test
+    public void testGetAllOrders() {
```

Comment:
Please use provide more complex test name, to understend what is tested, whith what condition and what the expected resut